solutions/account-infrastructure-base/variables.tf

```diff
@@ -19,6 +19,10 @@ variable "prefix" {
   type        = string
   description = "An optional prefix to append to all resources created by this solution."
   default     = null
+  validation {
+    condition     = length(var.prefix) <= 13
+    error_message = "`prefix` length must be 13 characters or less."
```

Comment:
where did you get 13 from? Suggest to leave a code comment explaining what module / service is restricting this